error: implicit declaration of function 'malloc' -Werror=implicit-function-declaration
   83 |  fade->original_buffer = malloc(size);
      |                          ^~~~~~[
../fade.c:83:26: error: incompatible implicit declaration of built-in function 'malloc' -Werror
../fade.c:8:1: note: include '<stdlib.h>' or provide a declaration of 'malloc'
    7 | #include <string.h>
  +++ | #include <stdlib.h>
    8 | 

diff --git a/fade.c b/fade.c
index 946209a..94cd47d 100644
--- a/fade.c
+++ b/fade.c
@@ -1,6 +1,7 @@
 #include "fade.h"
 #include "pool-buffer.h"
 #include "swaylock.h"
+#include <stdlib.h>
 #include <stdio.h>
 #include <omp.h>
 #include <stdalign.h>
